frontend/src/pages/home/Home.tsx

```diff
@@ -85,16 +112,23 @@ function Home() {
   return (
     <StyledContainer>
       <HomeHeader />
+      <StyledActionWrapper>
+        <StyledFilterWrapper>
+          <SpecialtyFilterModalButton handleOpenModal={handleOpenModal} />
+          <SpecialtyFilterModal
+            opened={modalOpened}
+            handleCloseModal={handleCloseModal}
+            selectedSpecialties={selectedSpecialties}
+            handleApplyFinalSpecialties={handleApply}
+          />
+          <SortButton handleSortButtonClick={handleSortButtonClick} />
+        </StyledFilterWrapper>
+        <Button onClick={handleMentoringCreation} customStyle={customSytle}>
+          {myMentoringId === null ? '멘토링 개설하기' : '멘토링 관리하기'}
+        </Button>{' '}
```

Comment:
p4: 공백 없어져도 괜찮을 거 같아 👍👍
```suggestion
        </Button>
```

Reply:
👍👍👍